### DISCUSSION TITLE
Change whitespace character

### DISCUSSION THREAD

**Discussion by fehmer** [monkeytypegame/monkeytype#4477] [Ideas]
Allow the user to pick the whitespace symbol e.g. `␣` or  `⸱`.

Examples:
![image](https://github.com/monkeytypegame/monkeytype/assets/3728838/8f3ba667-b218-48e1-909e-7dd3c6de8591)

![image](https://github.com/monkeytypegame/monkeytype/assets/3728838/121ca714-6966-4753-a44c-57d3a9ddb452)

Or even allow the user to provide his own:
![image](https://github.com/monkeytypegame/monkeytype/assets/3728838/c6022fcb-b4bc-4db2-ac95-276c4bb74f4d)

It there is interest for this feature I can create a PR for it.

**Comment by Miodec** [comment]
Not sure its worth adding. Whats the advantage?
